packages/base/src/UI5Element.js

```diff
@@ -779,6 +779,12 @@ class UI5Element extends HTMLElement {
 		if (!this.staticAreaItem) {
 			this.staticAreaItem = StaticAreaItem.createInstance();
 			this.staticAreaItem.setOwnerElement(this);
+			if (this.getStaticAreaItemStableDomRef) {
+				const stableDomRef = this.getStaticAreaItemStableDomRef();
+				if (stableDomRef) {
+					this.staticAreaItem.setAttribute("data-ui5-stable", stableDomRef);
+				}
+			}
```

Comment:
There needs to be a way to add a stable selector to the static area item of any web component. In this case we need it to be added to the shellbar because the module picker and overflow popovers need stable selectors.

Reply:
No, the stable selector needs to be on the `<ui5-static-area-item>` itself. The popup tag already got a stable dom ref in the hbs file, both tags need a stable dom ref.

The problem here is that the web assistant needs to be able to select every shadow document from the root all the way down to the contents in the menu.

Reply:
Here is a link to the line in this PR where the stable dom refs were already added to the popover elements:
https://github.com/SAP/ui5-webcomponents/pull/4401/files#diff-81fe9389e833266553a77f9a469631bbaa62744f6600b8df252ae1a8f33bd09eR3